src/System.Security.Cryptography.Algorithms/src/System/Security/Cryptography/IncrementalHash.net46.cs

```diff
@@ -0,0 +1,226 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.Security.Cryptography
+{
+    /// <summary>
+    /// Provides support for computing a hash or HMAC value incrementally across several segments.
+    /// </summary>
+    public sealed class IncrementalHash : IDisposable
+    {
+        private const int NTE_BAD_ALGID = unchecked((int)0x80090008);
+
+        private static readonly byte[] s_empty = new byte[0];
```

Comment:
This can be removed and any uses of `s_empty` can be replaced with `Array.Empty<byte>()`.